samples/core/src/main/resources/config/application-postgre.properties

```diff
@@ -0,0 +1,9 @@
+spring.jpa.database=postgresql
+spring.datasource.url=jdbc:postgresql://LIN34001298:5432/mydb?currentSchema=oasp4j
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL9Dialect
+spring.datasource.username=postgres
+spring.datasource.password=Postgre_123
```

Comment:
Never commit such things as username and password here. Only override this externally for development...